bibliopixel/led/circle.py

```diff
@@ -1,27 +1,26 @@
 from . base import LEDBase
 from .. import data_maker
+from . coord_map import gen_circle, calc_ring_steps, calc_ring_pixel_count
 
 
 class LEDCircle(LEDBase):
 
-    def __init__(self, drivers, rings, maxAngleDiff=0, rotation=0, threadedUpdate=False, masterBrightness=255, maker=data_maker.MAKER):
+    def __init__(self, drivers, rings=None, pixels_per=None,
+                 maxAngleDiff=0, rotation=0,
+                 threadedUpdate=False, masterBrightness=255, maker=data_maker.MAKER):
         super().__init__(drivers, threadedUpdate, masterBrightness, maker)
         self.rings = rings
         self.maxAngleDiff = maxAngleDiff
-        self._full_coords = False
+        full_coords = False
```

Comment:
It just is a flag for later to say if the rings passed in had ALL indices or just start/stop for each ring and needed to be resolved later. Honestly, this is a remnant of an old way of doing things and could be refactored out.